tests/admin_views/tests.py

```diff
@@ -6275,6 +6276,152 @@ def test_hidden_fields_small_window(self):
         finally:
             self.selenium.set_window_size(current_size["width"], current_size["height"])
 
+    def test_updating_related_objects_updates_fk_selects(self):
+        from selenium.webdriver.common.by import By
+        from selenium.webdriver.support.ui import Select
+
+        BORN_COUNTRY_SELECT_ID = "id_born_country"
+        LIVING_COUNTRY_SELECT_ID = "id_living_country"
+        FAVORITE_COUNTRY_TO_VACATION_SELECT_ID = "id_favorite_country_to_vacation"
+        CONTINENT_SELECT_ID = "id_continent"
```

Comment:
I see no reason to uppercase these variables :thinking:

Reply:
Just because I considered them as constants.

Updated!